lib/custom_switch.dart

```diff
@@ -51,67 +110,67 @@ class _CustomSwitchState extends State<CustomSwitch>
       animation: _animationController,
       builder: (context, child) {
         return GestureDetector(
-          onTap: () {
-            if (_animationController.isCompleted) {
-              _animationController.reverse();
-            } else {
-              _animationController.forward();
-            }
-            widget.value == false
-                ? widget.onChanged(true)
-                : widget.onChanged(false);
-          },
-          child: Container(
-            width: 70.0,
-            height: 35.0,
-            decoration: BoxDecoration(
-                borderRadius: BorderRadius.circular(20.0),
-                color: _circleAnimation.value == Alignment.centerLeft
-                    ? widget.inactiveColor
-                    : widget.activeColor),
-            child: Padding(
-              padding: const EdgeInsets.only(
-                  top: 4.0, bottom: 4.0, right: 4.0, left: 4.0),
-              child: Row(
-                mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                children: <Widget>[
-                  _circleAnimation.value == Alignment.centerRight
-                      ? Padding(
-                          padding: const EdgeInsets.only(left: 4.0, right: 4.0),
-                          child: Text(
-                            widget.activeText,
-                            style: TextStyle(
-                                color: widget.activeTextColor,
-                                fontWeight: FontWeight.w900,
-                                fontSize: 16.0),
-                          ),
-                        )
-                      : Container(),
-                  Align(
-                    alignment: _circleAnimation.value,
-                    child: Container(
-                      width: 25.0,
-                      height: 25.0,
-                      decoration: BoxDecoration(
-                          shape: BoxShape.circle, color: Colors.white),
-                    ),
+            onTap: () {
+              if (_animationController.isCompleted) {
+                _animationController.reverse();
+              } else {
+                _animationController.forward();
+              }
+              widget.onChanged(!widget.value);
+            },
+            child: Tooltip(
+              message: widget.value ? widget.activeTooltip : widget.inactiveTooltip,
```

Comment:
@jamiecash can you share a video/ss with tooltip? how's the switch looking with tooltip? All other changes looks fine to me.